diff --git a/main/CMakeLists.txt b/main/CMakeLists.txt
index d77ecc7..475d10d 100644
--- a/main/CMakeLists.txt
+++ b/main/CMakeLists.txt
@@ -265,6 +265,11 @@ elseif(CONFIG_BOARD_TYPE_XMINI_C3)
     set(BOARD_TYPE "xmini-c3")
     set(BUILTIN_TEXT_FONT font_noto_sans_basic_14_1)
     set(BUILTIN_ICON_FONT font_material_symbols_14_1)
+elseif(CONFIG_BOARD_TYPE_POCKET_WALL_E_C3)
+    set(BOARD_TYPE "pocket-wall-e-c3")
+    set(BOARD_NAME "pocket-esp-ai-by-huy-vector")
+    set(BUILTIN_TEXT_FONT font_noto_sans_basic_14_1)
+    set(BUILTIN_ICON_FONT font_material_symbols_14_1)
 elseif(CONFIG_BOARD_TYPE_ESP_KORVO2_V3)
     set(BOARD_TYPE "esp32s3-korvo2-v3")
     set(BUILTIN_TEXT_FONT font_noto_sans_basic_20_4)
diff --git a/main/Kconfig.projbuild b/main/Kconfig.projbuild
index a1bb6e3..bd718a9 100644
--- a/main/Kconfig.projbuild
+++ b/main/Kconfig.projbuild
@@ -153,6 +153,9 @@ choice BOARD_TYPE
     config BOARD_TYPE_XMINI_C3
         bool "Xmini C3"
         depends on IDF_TARGET_ESP32C3
+    config BOARD_TYPE_POCKET_WALL_E_C3
+        bool "Huy Vector Pocket Wall-E ESP32-C3"
+        depends on IDF_TARGET_ESP32C3
     config BOARD_TYPE_ESP_KORVO2_V3
         bool "Espressif Korvo2 V3"
         depends on IDF_TARGET_ESP32S3
